hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java

```diff
@@ -165,6 +164,13 @@ public MirrorContractResultResponse getContractResultByTransactionId(String tran
                 transactionId);
     }
 
+    public JsonRpcSuccessResponse contractsCallSimulation(String data, String to, String from) {
```

Comment:
Not sure why you're using the simulation terminology, but it should just be named like the URI path.
```suggestion
    public ContractCallResponse contractsCall(String data, String to, String from) {
```